README.md

```diff
@@ -26,20 +26,13 @@ The module handles the following account settings:
     - Restrict: traffic that originates outside the specified allowed IP addresses is blocked
 - Global shell settings (on - off)
 - Public access group (on - off)
+- Financial Services Validated products (on - off)
+- User List Visibility restrictions (on - off)
 
 The module supports creating and updating settings that are applied with the `terraform apply` command. With objects
 affected by the `destroy` command, the module preserves the most recent setting and doesn't change objects that are
 configured outside of Terraform's scope.
 
-### Current limitations:
-The module currently does not support setting the following FSCloud requirements:
-- Check whether user list visibility restrictions are configured in IAM settings for the account owner
-  - Follow these [steps](https://cloud.ibm.com/docs/account?topic=account-iam-user-setting) as a workaround to set this manually in the UI
-- Check whether the Financial Services Validated setting is enabled in account settings
-  - Follow these [steps](https://cloud.ibm.com/docs/account?topic=account-enabling-fs-validated) as a workaround to set this manually in the UI
-
-Tracking issue with IBM provider -> https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4204
```

Comment:
I would still leave this section here, however update to to say that the module is using the restapi provider as a workaround until the provider issue (link it) is addressed